drivers/can/can_handlers.c

```diff
@@ -138,6 +138,14 @@ static inline int z_vrfy_can_get_max_filters(const struct device *dev, enum can_
 }
 #include <syscalls/can_get_max_filters_mrsh.c>
 
+static inline int z_vrfy_can_get_capabilities(const struct device *dev, can_mode_t *cap)
+{
+	Z_OOPS(Z_SYSCALL_DRIVER_CAN(dev, get_capabilities));
+
```

Comment:
We should make sure the thread has write access.
`Z_OOPS(Z_SYSCALL_MEMORY_WRITE(cap, sizeof(*cap))); `

Reply:
Nice catch, forgot about that. Added.